projects/packages/my-jetpack/_inc/components/connected-product-card/index.jsx

```diff
@@ -33,7 +33,7 @@ const ConnectedProductCard = ( { admin, slug, showDeactivate } ) => {
 			name={ name }
 			description={ description }
 			status={ status }
-			icon={ <Icon /> }
+			icon={ <Icon opacity={ 0.4 } /> }
```

Comment:
why are we handling the opacity by attr instead of via CSS?

Reply:
For the same reason we're handling the color via attribute